src/Pages/Music/Album/Albumid_Int.elm

```diff
@@ -133,59 +151,83 @@ view model =
-                                [ Element.text "Genre: "
+                            ]
+                        , column [ alignTop, Element.height fill, Element.width (fillPortion 7), paddingXY 10 25 ]
+                            [ row [] [ el [ Font.color white, Font.size 30 ] (Element.text album.label), el [ alignBottom, paddingXY 10 0, Font.size 15 ] (Element.text (String.fromInt album.year)) ]
+                            , column [ paddingXY 0 35, spacingXY 0 10, Font.size 14 ]
+                                [ row []
+                                    [ el [ Font.color white ] (Element.text "Artist(s): ")
+                                    , row []
+                                        (List.map
+                                            (\artist ->
+                                                Element.text artist
+                                            )
+                                            album.artist
+                                        )
+                                    ]
                                 , row []
-                                    (List.map
-                                        (\genre ->
-                                            Element.text genre
+                                    [ el [ Font.color white ] (Element.text "Genre: ")
+                                    , row []
+                                        (List.map
+                                            (\genre ->
+                                                Element.text genre
+                                            )
+                                            album.genre
                                         )
-                                        album.genre
-                                    )
+                                    ]
+                                ]
+                            , row [ spacingXY 10 0 ]
+                                [ Input.button [ paddingXY 12 8, Background.color Colors.navTextHover ]
+                                    { onPress = Just PlayMsg
+                                    , label = row [] [ el [ Font.color white, paddingEach { top = 0, left = 0, right = 10, bottom = 0 } ] (Element.text "Play"), Element.html (Filled.play_circle_filled 16 (MITypes.Color <| whiteIcon)) ]
+                                    }
+                                , Input.button [ paddingXY 12 8, Background.color (Element.rgba255 71 74 75 1) ]
+                                    { onPress = Just QueueMsg
+                                    , label = row [] [ el [ Font.color white, paddingEach { top = 0, left = 0, right = 10, bottom = 0 } ] (Element.text "Queue"), Element.html (Filled.add_circle 16 (MITypes.Color <| greyIcon)) ]
+                                    }
+                                , Input.button [ paddingXY 12 8, Background.color (Element.rgba255 71 74 75 1) ]
+                                    { onPress = Just PlayMsg
```

Comment:
This and the next one are placeholders for now? I guess you could set `Nothing` instead and maybe a todo comment?

Reply:
I thought, that there should probably be a TODO in front of the comment, so that we can find it by search later.